src/main/java/com/powsybl/openloadflow/sa/WoodburyDcSecurityAnalysis.java

```diff
@@ -89,8 +90,7 @@ private DenseMatrix calculatePostContingencyStatesForAContingency(DcLoadFlowCont
 
             // if a phase tap changer is lost or if the connectivity have changed, we must recompute load flows
             if (!disabledBuses.isEmpty() || !lostPhaseControllers.isEmpty()) {
-                double[] newFlowStatesArray = DcSensitivityAnalysis.runDcLoadFlow(loadFlowContext, disabledNetwork, reportNode);
-                newFlowStates = new DenseMatrix(newFlowStatesArray.length, 1, newFlowStatesArray);
+                newFlowStates = DcSensitivityAnalysis.runDcLoadFlow(loadFlowContext, disabledNetwork, reportNode);
```

Comment:
This dependency to DcSensitivityAnalysis is not allowed.

Reply:
Same, here is a PR dedicated to this issue: https://github.com/powsybl/powsybl-open-loadflow/pull/1082